source/UIAHandler/__init__.py

```diff
@@ -572,9 +702,8 @@ def IUIAutomationEventHandler_HandleAutomationEvent(self,sender,eventID):
 			else:
 				if _isDebug():
 					log.debugWarning(
-						"HandleAutomationEvent: Dropping textChange event from element "
-						f"with class name {repr(sender.currentClassName)} "
-						f"and automation ID {repr(sender.CachedAutomationID)}"
+						f"HandleAutomationEvent: Dropping textChange event "

```

Comment:
Remove unnecessary f string

```suggestion
						"HandleAutomationEvent: Dropping textChange event "
```